WallEView/ViewController.swift

```diff
@@ -103,13 +113,14 @@ class ViewController: NSViewController, NSTableViewDataSource, NSTableViewDelega
             DispatchQueue.main.async {
                 if let data = data {
                     do {
-                        let state = try JSONDecoder().decode(State.self, from: data)
-                        self?.state = state
+                        let queues = try JSONDecoder().decode([Queue].self, from: data)
```

Comment:
@AliSoftware adjusted parsing here, for now just filtering everything that is no targeted to develop branch. In future I think we can add a drop down selected for user to select target branch and to switch between different queues. what do you think?

Reply:
Reading your code it's more about "filtering everything that is not the _first_ queue. Agreed that since most of our target branches are either `develop` or `release/*`, then `develop` is likely to be the first queue (they are ordered alphabetically in the JSON API response), but what if Ben creates a branch `ben/xxx-123` then want to split his work and have a second PR that builds on top of the first one and thus makes a PR from `ben/xxx-123-bis` branch to `ben/xxx-123` branch… then this new target branch will be the `.first` though 😉 

Maybe it's worth adding a convenience method `var developQueue: Queue? { return queues.filter { $0.targetBranch == "develop" }` on `State` and use that instead of `queues.first` throughout your changes here?

Reply:
I think it's ok to filter just develop for now, as soon as I add branch selector we won't need this code, and we won't need `developQueue` property as well